raiden/tests/integration/fixtures/transport.py

```diff
@@ -69,8 +69,9 @@ def matrix_transports(
                 'server': server,
                 'server_name': server.netloc,
                 'available_servers': local_matrix_servers,
-                'private_rooms': private_rooms,
-            }),
+                'private_rooms': privacy_settings[transport_index] if
+                                 type(privacy_settings) == list else privacy_settings,
```

Comment:
I don't like these mixed types for a single fixture. The fixture should always return a list of len() `number_of_transports`.